tests/test_nodes.py

```diff
@@ -656,7 +656,9 @@ def test_get(self):
         )
 
         assert _call("Node0_L6_Y", properties=[Cell.X, Cell.MTYPE, Cell.LAYER]).empty
-        assert _call(1, properties={Cell.MTYPE}).tolist() == ["L6_Y"]
+        assert _call(1, properties=[Cell.MTYPE]).tolist() == ["L6_Y"]
+        assert _call([1], properties=Cell.MTYPE).tolist() == ["L6_Y"]
+        assert _call([1, 2], properties=Cell.MTYPE).tolist() == ["L6_Y", "L6_Y"]
```

Comment:
Passing a dict as `properties` causes a warning in pandas:
> FutureWarning: Passing a set as an indexer is deprecated and will raise in a future version. Use a list instead.

I removed the previous test because the `properties` parameter in the `get` method is expected to be a list or a str (as explained in the dostring), not a dict. If a dict is used, it still works with the current version of pandas, but a warning is printed (by pandas itself).

I added a test passing (list of ids, properties=str).

Reply:
Ok, cool 👍